docs/generated/cmd/kumactl/kumactl_install_observability.md

```diff
@@ -0,0 +1,41 @@
+## kumactl install observability
+
+Install Observability (Metrics, Logging, Tracing) backend in Kubernetes cluster (Prometheus + Grafana + Loki + Jaeger + Zipkin)
+
+### Synopsis
+
+Install Observability (Metrics, Logging, Tracing) backend in Kubernetes cluster (Prometheus + Grafana + Loki + Jaeger + Zipkin) in its own namespace.
+
+```
+kumactl install observability [flags]
+```
+
+### Options
+
+```
+  -h, --help                        help for observability
+      --jaeger-address string       the address of jaeger to query (default "http://jaeger-query.kuma-observability")
+      --kuma-cp-address string      the address of Kuma CP (default "http://kuma-control-plane.kuma-system:5676")
+      --loki-address string         the address of the loki to query (default "http://loki.kuma-observability:3100")
+  -m, --mesh string                 mesh to use (default "default")
+      --namespace string            namespace to install observability to (default "kuma-observability")
+      --prometheus-address string   the address of the prometheus server (default "http://prometheus-server.kuma-observability")
+      --without-grafana             disable Grafana resources generation
+      --without-jaeger              disable Jaeger resources generation
+      --without-loki                disable Loki resources generation
+      --without-prometheus          disable Prometheus resources generation
```

Comment:
how about: `--components` with default value of `grafana, jaeger, loki, prometheus`?